src/Widgets/DataFormBooleanWidget/DataFormBooleanWidgetComponent.tsx

```diff
@@ -15,27 +15,40 @@ provideComponent(DataFormBooleanWidget, ({ widget }) => {
   const labelOptions: { htmlFor?: string } = {}
   if (!isInPlaceEditingActive()) labelOptions.htmlFor = id
 
+  const className =
+    widget.get('style') === 'switch' ? 'form-check form-switch' : 'form-check'
+
+  const submitOnChange = widget.get('submitOnChange')
```

Comment:
Why is `submitOnChange` a property of the `BooleanWidget`?
IMHO it should be a property of the `FormWidget`.

Because having a form where some fields are "submitOnChange" and others are not would be confusing.

Let's keep it simple for the editor and say "a form either submits on change, or i does not".

Or am I missing something?

Reply:
Good idea: My initial idea was, that only a `DataFormBooleanWidget` without a container could be used. But that would duplicate the save logic to `DataFormBooleanWidget`.

So yeah, let's move `submitOnChange` up to the `DataFormContainerWidget`.